install/windows/docker-ee.md

```diff
@@ -1,23 +1,35 @@
 ---
-description: How to install Docker EE for Windows Server
-keywords: Windows, Windows Server, install, download, ucp, Docker EE
-title: Install Docker Enterprise Edition for Windows Server
+description: How to install Docker Engine - Enterprise for Windows Server
+keywords: Windows, Windows Server, install, download, ucp, Docker Engine - Enterprise
+title: Install Docker Engine - Enterprise on Windows Servers
 redirect_from:
 - /docker-ee-for-windows/install/
 - /engine/installation/windows/docker-ee/
 ---
 
 {% capture filename %}{{ page.win_latest_build }}.zip{% endcapture %} {% capture download_url %}https://download.docker.com/components/engine/windows-server/{{ site.docker_ee_version }}/{{ filename }}{% endcapture %}
 
-Docker Enterprise Edition for Windows Server (*Docker EE*) enables native Docker containers on Windows Server. Windows Server 2016 and later versions are supported. The Docker EE installation package includes everything you need to run Docker on Windows Server.  This topic describes pre-install considerations, and how to download and install Docker EE.
+Docker Engine - Enterprise enables native Docker containers on Windows Server. Windows Server 2016 and later versions are supported. The Docker Engine - Enterprise installation package includes everything you need to run Docker on Windows Server.  This topic describes pre-install considerations, and how to download and install Docker Engine - Enterprise.
 
 > Release notes
 >
 > [Release notes for all versions](/release-notes/)
 
-## Install Docker EE
+## System requirements
 
-Docker EE for Windows requires Windows Server 2016 or later. See
+Windows OS requirements around specific CPU and RAM requirements also need to be met as specified 
+in the [Windows Server Requirements](https://docs.microsoft.com/en-us/windows-server/get-started/system-requirements).
+This provides information for specific CPU and memory specs and capabilities (instruction sets like CMPXCHG16b, 
+LAHF/SAHF, and PrefetchW, security: DEP/NX, etc.).
+
+* OS Versions: Server 2016 (Core and GUI), 1709 and 1803 and soon Server 2019 (Core and GUI)
```

Comment:
Let's remove the `soon Server 2019 (Core and GUI)` till it's officially out and GA. UCP/EE does not support Server 2019 yet as far as I know. It's being tested. My note around it being soon was more of a FYI.

Reply:
done.